ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -218,6 +218,7 @@ CLUSTER_NAME=$(<"${SHARED_DIR}/cluster_name")
 yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" - <<< "
 apiVersion: v1
 baseDomain: ${BASE_DOMAIN}
+fips: true
```

Comment:
Thsi is hardcoded, so all of our jobs will be FIPS mode.

Please introduce it as an environment variable, so that in the job configuration we can control the value. 

In `baremetal-lab-agent-install-ref.yaml`, add
```
  - name: FIPS_ENABLED
    default: "false"
    documentation: "Install a cluster with FIPS mode"
```

Reply:
`fips: ${FIPS_ENABLED}`

Note I used the variable name `FIPS_ENABLED`, this is consistent with the rest install jobs from cors installer.

Reply:
Maybe, this is already done by the [ipi-conf](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh#L31-L36) step that we should execute for both agent, upi and bm ipi.  setting FIPS_ENABLED to true in the config might be enough.

Reply:
Thanks, you are right, we already have it https://steps.ci.openshift.org/chain/baremetal-lab-agent-conf.

@zniu1011 then there is no need to configure the install-config.yaml here. Just overwrite the env from the job. Here is a random example

```
- as: aws-c2s-ipi-disconnected-private-fips-f14
  ...
  steps:
    ...
    env:
      ...
      FIPS_ENABLED: "true"
```